src/models/service/addresses.rs

```diff
@@ -0,0 +1,28 @@
+use serde::Serialize;
+
+#[derive(Serialize, Debug, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct AddressEx {
+    pub value: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub name: Option<String>,
```

Comment:
Ah ok, so we were exposing this as nullable from the past. This answers the above question :point_up:

Reply:
We had the `AddressInfo` (what is returned from the backend) and there it is an empty string instead of null. But null makes more sense here